### PR TITLE
Adding yamllinting github action and some minor syntax corrections in yml-files. 

### PR DESCRIPTION
**Pull request by fslds**
## TL;DR

Added yamllinting to the repository to ensure yaml files are valid. Should help contributors and maintainers. 

## PR purpose
While attempting to parse the LOLBAS yml files, I noticed that the template file had invalid syntax (special chars for the handles), which I corrected. 

In order to tackle the whole LOLBAS dataset, I added yamllinting configuration to the repository - both for local linting as well with GitHub actions. 

### GitHub Action
The benefit of the GitHub action is that it can serve as a test when new contributions are made, to check if the files are actually 'valid' yml files. Both the contributor as the maintainer then have ample opportunity to make any adjustments before the file is added to the `master` branch. Overall It should lighten the load on maintainer. 

in the project root diectoryr, a yamllint config was added: 
`.yamllint`: 
```yaml
---
extends: default
yaml-files:
  - '*.yml'
rules:
  new-line-at-end-of-file:
    level: warning
  trailing-spaces:
    level: warning
  line-length:
    level: warning
  new-lines:
    level: warning
  indentation:
    level: warning
```
It instructs yamllint to ignore excess whitespaces, overly long lines, and indentation issues, as here are plenty of them and they're not really breaking any 'code' to my knowledge.  In addition it will ignore differences between Windows and Unix-style newlines. 
Special, unescaped, characters will be caught, which are breaking for some yaml implementations such as pyyaml. 

The action is configured in `.github/workflows/yamllinting.yml`
```yaml
---
name: Yaml Lint
on: [push, pull_request]
jobs:
  lintFiles:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v1
      - name: yaml-lint
        uses: ibiqlik/action-yamllint@v3
        with:
          config_file: .yamllint
```
I compared two GitHub yaml linters, but neither provide per file output. So in case of any errors, the user would need to run local linting in order to get more details. It is only a health check at this point. 

### Local Linting

For local linting, you can issue the following command in the root of the repository.  It uses the same configuration as the GitHub action. 
```console
LOLBAS on feat/yamllint;
$ yamllint -c .yamllint '.'
```

which will give the following output: 
```
...
./yml/OSScripts/Manage-bde.yml
  7:81      warning  line too long (98 > 80 characters)  (line-length)
  8:81      warning  line too long (110 > 80 characters)  (line-length)
  14:81     warning  line too long (81 > 80 characters)  (line-length)
  15:81     warning  line too long (144 > 80 characters)  (line-length)
  16:81     warning  line too long (129 > 80 characters)  (line-length)
  22:81     warning  line too long (81 > 80 characters)  (line-length)

./.github/workflows/yamllint-compare.yml
  1:1       warning  missing document start "---"  (document-start)
  2:1       warning  truthy value should be one of [false, true]  (truthy)
  18:48     warning  no new line character at the end of file  (new-line-at-end-of-file)
```

Warnings can easily be suppressed by `--no-warnings`, rendering in no output:
```console
$ yamllint -c .yamllint . --no-warnings  
< no output> 
LOLBAS on  feat/yamllint; took 3s
 ```
### Syntax corrections
In addition I removed the 'new document' dashes `---` at the bottom of the template file, which indicates a new document and I think this repository represents only one document per file (and having two 2x dashes signals yaml that there are two, In theory the ending string of yaml is `...`. )
This is also present in most of the yml files as a consequence, but projects such as Jekyll do not care. 
In pyyaml you need to instruct it to just load the first document only, so the issue is easily circumvented. 

And there were a bunch of unescaped special characters in the yml files, and at one point a missing 'commands' key, if I recall it right, corrected those as well. 

## Future
I might revisit this in the future and add additional checking to make sure each yml file has the required fields and format, making it even easier to contribute new ymls.

### CODE REVIEW

**Comment by wietze** [comment]
Great idea, see also #122 for more YAML fixes. Would be nice to merge these two PRs together

**Comment by fslds** [comment]
@wietze Oh I went semi-blindly in, and didn't see your PR! Otherwise I would probably have used it as a base. 
If you want, I can add the yamllint part in a new branch based on yours, merge those first (in your fork), and have it as one PR (yours) as a whole. 

It will probably avoid conflicts as we both edited the same yaml files now.

**Comment by wietze** [comment]
That works for me, hopefully that will make it easier to review for @api0cradle 👍
Really liking this PR btw, another step in the right direction
